cpg_workflows/jobs/align.py

```diff
@@ -64,7 +62,7 @@ def _get_cram_reference_from_version(cram_version) -> str:
     if cram_version in cram_version_map:
         return cram_version_map[cram_version]
     raise ValueError(
-        f'Unrecognised cram_version: "{cram_version}", expected one of: {", ".join(cram_version_map.keys())}'
+        f'Unrecognised cram_version: "{cram_version}", expected one of: {", ".join(cram_version_map.keys())}',
```

Comment:
I guess this is emblematic of my concerns with automated formatting…

We (and everyone else) only ever give ValueError one argument, but the multi-line formatting the autoformatter forces us to use means that we are obliged to give it a misleading trailing comma.

I had to look it up to check that this didn't turn the arguments into a tuple undesirably. (Turns out a trailing comma in a Python function call is a special case and doesn't turn the argument into a tuple.)

\<le big sigh>

Reply:
I think this is a reasonable tradeoff for reduced diffs later on, same for lists, dicts, sets, etc.

Reply:
It's a reasonable tradeoff for things that do actually take a variable number of items.

It's suboptimal for things that take exactly one argument. Same with `dedent()` further on.